lib/linux_admin/network_interface/network_interface_rh.rb

```diff
@@ -162,7 +162,9 @@ def apply_static6(ip, prefix, gw, dns, search = nil)
     def save
       old_contents = File.read(@interface_file)
 
-      return false unless stop
+      stop_success = stop
+      stop_success = stop unless stop_success
+      return false unless stop_success
```

Comment:
After configure both IPv4 and IPv6 as dhcp. The dhclient instance for IPv6 exits and leaves a file in /var/run/dhclient6-eth0.pid. Now if we run  `ifdown eth0` the command fails because ifdown-eth is failing to kill a process that is no-longer running. It will then delete that pid file. So in the second run `ifdown eth0` it will return 0 and success.

See also: 
https://bugzilla.redhat.com/show_bug.cgi?id=1472396